### PR TITLE
Upgrade `shivammathur/setup-php`

### PR DESCRIPTION
**Pull request by GrahamCampbell**

### CODE REVIEW

File: .github/workflows/checks.yml [hunk 1]
```diff
@@ -9,14 +9,14 @@ jobs:
...
-        uses: shivammathur/setup-php@2.7.0
+        uses: shivammathur/setup-php@2.13.0
         with:
           php-version: 8.0
           coverage: none
           tools: composer-normalize
 
       - name: Checkout code
-        uses: actions/checkout@v2
+        uses: actions/checkout@V2
```
Comment (Nyholm) [inline .github/workflows/checks.yml:19]: It should be `actions/checkout@v2`

Comment (GrahamCampbell) [reply .github/workflows/checks.yml:19]: oops

Comment (GrahamCampbell) [reply .github/workflows/checks.yml]: fixed

**Comment by Nyholm** [comment]
It seams like phpstan has some internal errors. Im not sure why.

**Comment by GrahamCampbell** [comment]
Looks OK to me. Did you re-run them?

**Comment by jderusse** [comment]
> Looks OK to me. Did you re-run them?

I did

**Comment by Nyholm** [comment]
What is the added features in 2.13 compared to 2.7? Ie, is there something that effect us? 

Should we just do `shivammathur/setup-php@v2`?

**Comment by GrahamCampbell** [comment]
> Should we just do shivammathur/setup-php@v2?

Yes. :P

**Comment by Nyholm** [comment]
Could you rebase this so the ci is green?

**Comment by GrahamCampbell** [comment]
@Nyholm done. :)

**Comment by jderusse** [comment]
Thank you

**Pull request merged**